lib/operators/input.coffee

```diff
@@ -24,6 +24,8 @@ class Insert extends Operator
           @typedText,
           normalizeLineEndings: true
         )
+        cursor = @editor.getLastCursor()
+        cursor.moveLeft() unless cursor.isAtBeginningOfLine()
```

Comment:
Can you explain why you added these two lines? I'm thinking we should avoid operating only on the editor's *last* cursor, to make sure that these commands work smoothly with multiple cursors.